src/vm/exceptionhandling.cpp

```diff
@@ -3639,6 +3668,20 @@ ExceptionTracker* ExceptionTracker::GetOrCreateTracker(
                                            pExceptionRecord,
                                            pContextRecord);
 
+        if (fIsInterleavedHandling && fTransitionFromSecondToFirstPass)
+        {
+            // In interleaved exception handling a single exception is handled in an interleaved
+            // series of first and second passes. When we create a new tracker as a result
+            // of switching from the 2nd pass back to the 1st pass, we need to carry over
+            // several members that were set when processing one of the previous frames.
+            pNewTracker->m_sfResumeStackFrame = previousTrackerResumeStackFrame;
+            pNewTracker->m_uCatchToCallPC = previousTrackerCatchToCallPC;
+            pNewTracker->m_pClauseForCatchToken = previousTrackerClauseForCatchToken;
```

Comment:
Do we really need all these to be copied over? m_uCatchToCallPC, m_pClauseForCatchToken and m_ClauseForCatch  should not be needed, I think.